### ISSUE TITLE
code for files in data_base

### ISSUE THREAD

**Issue by 201younghanlee** [paulhager/MMCL-Tabular-Imaging#3]
Hi,
could you provide the code you used for preparing "labels_model_all_train_all_views.pt" in data_base folder (config)?

I having trouble reproducing your work.
I think it might has something to do with preparing the training dataset.

Also, any reason you set "nclasses = hparams.batch_size" in line 33 of models/MultimodalSimCLR.py ?
Are we only meant not to have more than one sample of the same class in a batch? 

Thanks

**Comment by paulhager** [comment]
Hi, I've added a new folder data that has my DVM dataset splits and the code to generate. Take a look and let me know if that helps. Its not particularly cleaned up but has everything you should need. 

I set nclasses to batch_size there because the nclasses referred to there is used for the calculation of the accuracy of the embedding matching task and so there are as many classes as you have subjects in your batch.

Gonna close this for now, feel free to make another issue if you have further questions or doubts.
